src/raft.c

```diff
@@ -341,6 +366,9 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
         return;
     }
 
+    void *client_session = getClientSession(rr, cmds);
+    if (client_session) {} /* avoid unused variable check */
```

Comment:
```suggestion
    (void) client_sessions;
```
This should also work, just fyi.